ebuild-writing/file-format/text.xml

```diff
@@ -177,9 +179,18 @@ exact copy of the contents of
 header.txt</uri></c> in the top directory of the Gentoo repository.
 </p>
 
+<p>
+It is good style to have one blank line after the EAPI line and another blank
+line after any inherits.
+</p>
```

Comment:
This doesn't belong here, because it is not related to the ebuild header.

Reply:
Can you please suggest where this (and the other item) _should_ go?

Reply:
Not in this chapter. ebuild-writing/using-eclasses may be a good place.